client/state/jetpack-connect/reducer/test/jetpack-auth-attempts.js

```diff
@@ -26,9 +20,9 @@ describe( '#jetpackAuthAttempts()', () => {
 				attemptNumber: 2,
 			}
 		);
-		expect( state[ 'example.com' ] )
-			.to.have.property( 'timestamp' )
-			.to.be.at.least( nowTime );
+		expect( state[ 'example.com' ] ).toMatchObject( {
+			timestamp: expect.any( Number ),
```

Comment:
Previously we were verifying that it's not a past date here. Should we continue that, or do you consider it's not necessary?

Reply:
I think it adds very little safety, but I'm happy to restore it.

Reply:
I agree it's not very useful, so I'm leaving it to your judgement.